src/translations/en.json

```diff
@@ -421,6 +421,13 @@
       "relative_time": {
         "never": "Never",
         "just_now": "Just now",
+        "abbreviated_duration": {
+          "second": "{count}s",
+          "minute": "{count}m",
+          "hour": "{count}h",
+          "day": "{count}d",
+          "week": "{count}w"
+        },
```

Comment:
I think there should a be space between value and unit to be consistent with the rest of Lovelace.

Reply:
Personally, I think that would look off and don't believe this should necessarily conform to the rest of Lovelace.